components/automate-ui/src/app/entities/service-groups/service-groups.model.ts

```diff
@@ -19,6 +19,7 @@ export interface Service {
   fqdn: string;
   site: string;
   channel: string;
+  update_strategy: string;
```

Comment:
This could use a string literal type, I guess? `update_strategy: "NONE" | "ROLLING" | "AT_ONCE";`

Reply:
FYI: It is `AT-ONCE`, but I like that idea!

Reply:
Yeah, you get the gist 😉

Reply:
![tenor-70491942](https://user-images.githubusercontent.com/5712253/60970437-44343b00-a322-11e9-97dd-8ad411b8fc1e.gif)